src/building/building.h

```diff
@@ -75,11 +93,11 @@ typedef struct building {
             short queued_docker_id;
             unsigned char num_ships;
             signed char orientation;
-            short docker_ids[3];
+            short cartpusher_ids[3];
             short trade_ship_id;
             unsigned char has_accepted_route_ids;
             int accepted_route_ids;
-        } dock;
+        } distribution;
```

Comment:
My bad, I wasn't clear about this. I would keep the `dock` struct but move the old `docker_ids` to a new `cartpusher_ids` in a new `distribution` struct.